files/es/web/javascript/reference/classes/extends/index.md

```diff
@@ -58,8 +58,23 @@ class myDate extends Date {
   }
 
   getFormattedDate() {
-    var months = ['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'];
-    return this.getDate() + "-" + months[this.getMonth()] + "-" + this.getFullYear();
+    var months = [
+      "Jan",
+      "Feb",
+      "Mar",
+      "Apr",
+      "May",
+      "Jun",
+      "Jul",
+      "Aug",
+      "Sep",
+      "Oct",
+      "Nov",
+      "Dec",
+    ];
+    return (
```

Comment:
I know prettier didn't complain about the space but I'd like to see a space between them. what do you think?
```suggestion
    ];

    return (
```